trs_filer/api/additions.openapi.yaml

```diff
@@ -548,15 +548,38 @@ components:
             Other values MAY be used, as long as implementors are aware of the issues discussed in https://tools.ietf.org/html/rfc6920#section-9.4[RFC6920].
 
             GA4GH may provide more explicit guidance for use of non-IANA-registered algorithms in the future.
+    TypeRegister:
```

Comment:
Did `anyOf` not work?

Reply:
I kinda get a bad feeling to see file types of different categories lumped together, like apples and oranges. But I suppose we can disentangle that together with the other type information (`PRIMARY_DESCRIPTOR` etc.).